src/OpenCL/Extensions/Silk.NET.OpenCL.Extensions.INTEL/Enums/AdvancedMotionEstimationIntraSearchPredictionModesBufferChromaBlock.gen.cs

```diff
@@ -9,9 +9,12 @@
 
 namespace Silk.NET.OpenCL.Extensions.INTEL
 {
+    [Flags]
     [NativeName("Name", "cl_intel_advanced_motion_estimation.intra_search_prediction_modes_buffer.chroma_block")]
     public enum AdvancedMotionEstimationIntraSearchPredictionModesBufferChromaBlock : int
     {
+        [NativeName("Name", "")]
```

Comment:
this enum shouldnt be a `[Flags]`, and shouldnt have a `None` member, given it already has one with a value of 0